test/parallel/test-stream2-readable-wrap.js

```diff
@@ -52,19 +49,18 @@ function runTest(highWaterMark, objectMode, produce) {
     }
   }
 
-  var w = new Writable({ highWaterMark: highWaterMark * 2,
+ const w = new Writable({ highWaterMark: highWaterMark * 2,
                          objectMode: objectMode });
-  var written = [];
+  const written = [];
   w._write = function(chunk, encoding, cb) {
     console.log('_write', chunk);
     written.push(chunk);
-    setTimeout(cb);
+    setTimeout(cb, 1000);
```

Comment:
Can you change this to `1` instead of `1000`? No duration specified defaults to 1ms, so using `1` keeps the behavior identical but makes things more explicit.